src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -224,6 +222,13 @@ export class DevWorkspaceTasks {
           task.title = await `${task.title}...OK`
         },
       },
+      {
+        title: 'Delete DevWorkspace Namespace',
```

Comment:
Do you mean DevWorkspace Operator namespace here? If so, please make it more clear.

Reply:
I'll do fix it

Reply:
fixed.